public/locales/en/zodCustom.json

```diff
@@ -1,7 +1,12 @@
 {
     "custom_error": "This is a custom error.",
+    "password_complexity": "password needs to be at least 6 characters long",
     "custom_error_with_path": "'{{- path}}' is not valid. This is a custom error.",
     "paths": {
-        "customErrorInline": "The Translated Field Name"
+        "customErrorInline": "The Translated Field Name",
+        "phone": "Invalid Phone Number",
```

Comment:
I don't like this. 'paths' is only to translate the name of the field, not to specify the problem.

Reply:
this was a leftover